skimage/measure/tests/test_fit.py

```diff
@@ -41,8 +42,12 @@ def test_line_model_nd_invalid_input():
     with testing.raises(ValueError):
         LineModelND().predict_y(np.zeros(1), np.zeros(1))
 
-    assert not LineModelND().estimate(np.empty((1, 3)))
-    assert not LineModelND().estimate(np.empty((1, 2)))
+    tf = LineModelND.from_estimate(np.empty((1, 3)))
+    assert not tf
+    assert str(tf) == 'LineModelND: estimate under-determined'
+    tf = LineModelND.from_estimate(np.empty((1, 2)))
```

Comment:
Nitpick: it shouldn't matter for this case, but might still be defensive practice to use something determined instead of `np.empty`?
```suggestion
    tf = LineModelND.from_estimate(np.ones((1, 3)))
    assert not tf
    assert str(tf) == 'LineModelND: estimate under-determined'
    tf = LineModelND.from_estimate(np.ones((1, 2)))
```